src/elements/content-sidebar/ActivitySidebar.js

```diff
@@ -1193,6 +1195,7 @@ class ActivitySidebar extends React.PureComponent<Props, State> {
             onTaskView,
         } = this.props;
         const { activityFeedError, approverSelectorContacts, contactsLoaded, mentionSelectorContacts } = this.state;
+        const newThreadedRepliesEnabled = isFeatureEnabled(features, 'activityFeed.newThreadedReplies.enabled');
```

Comment:
minor: can we rename it to something like `isNewThreadedRepliesEnabled`? I think for most of the other boolean values we follow this naming convention